### DISCUSSION TITLE
Auth - macOS menu bar icon in monochrome or ability to hide

### DISCUSSION THREAD

**Discussion by marcolaux** [ente-io/ente#2410] [Enhancements]
Currently the macOS menu bar icon does not blend with the rest of the aesthetics:

<img width="358" alt="image" src="https://github.com/ente-io/ente/assets/6410852/10b2285e-f4d9-4624-b621-d620f99c1b92">

It would be nice to have a monochrome menu bar icon or an ability to hide it.

I don't want to quit the app all the time to hide it as I'm a web dev and need an auth app open during my workday.

It can be distracting if you have a short or weird attention span and when you can get distracted by things easily that are off.

**Comment by Moulick** [comment]
Definitely needed, specially the ability to hide. And even more important for MacBooks with the notch, the menu bar real estate is at a premium!

**Comment by ninjada** [comment]
is this possible yet?

im doubly confused cause im running same 4.00 version on two different macs and only one is plagued with this icon.....

**Comment by degritsenko** [comment]
+1

**Comment by paragpindoria** [comment]
`I don't want to quit the app all the time to hide it as I'm a web dev and need an auth app open during my workday.`

See https://github.com/ente-io/ente/discussions/4501

Slightly different take on this, if the application is within the pop-up of the menu bar, rather than the current 'Show Window' and 'Hide Window' then it would be streamlined as it is one click rather than multiple. Also, you won't have application window left open which reduces clutter.

**Comment by delro92** [comment]
After a little bit over a year, good news incoming: https://github.com/ente-io/ente/commit/4f224e7eba4782bf7d12661db687bbf2e6800e5f
